Makefile

```diff
@@ -126,3 +126,14 @@ setup-synocommunity: setup
 		-e "s|REPORT_URL=.*|REPORT_URL=https://github.com/SynoCommunity/spksrc/issues|" \
 		local.mk
 
+all-affected-spks-from-master:
+	for spk in $(git diff --stat upstream/master...HEAD | bash mk/build-target-determinator.bash | sort) ; \
```

Comment:
Sorry, I force-pushed a rebase rather than stack a commit below this.

Previously, this was just a `for spk in $(bash mk/build...); \` and did the `git diff` inside the script.  The suggestion on that was to look for a common revision between `master` branch and the PR branch, and determine the changed spikes from that.  As you can see, this script now does that.